.changeset/stupid-plants-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@blobscan/web": patch
```

Comment:
should be minor at least

Reply:
Done [a25a27d](https://github.com/Blobscan/blobscan/pull/493/commits/a25a27daa36b17b956d79be9fef9a051c551ec07)